cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -144,7 +155,26 @@ func generateMountsModeInstall() error {
 		return nil
 	}
 
-	// 4. done, no output, no error indicates to initramfs we are done
+	// 4. write $(ubuntu_data)/var/lib/snapd/modenv
+	_, err = boot.ReadModeenv(filepath.Join(runMnt, "ubuntu-data"))
```

Comment:
ubuntu-data was a just created tmpfs on the last invocation, do we need to really check if the file is there?

Reply:
Good point, as long as this is the last step of the chain of things that run I can omit this check.